site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -0,0 +1,61 @@
++++
+date = "2017-08-16T14:31:00-04:00"
+title = "Citra Progress Report - 2017 July"
+tags = [ "progress-report" ]
+author = "saphiresurf"
+forum = 0000
++++
+
+With June bringing in many huge improvements to GPU functionality, July has come in to sweep up and improve accuracy in a few distinct areas of Citra. Changes and fixes ranging from texture handling all the way to aesthetic changes that help improve the visual appeal and customizability of Citra's Qt interface. Now, let's get into what's changed!
+
+## [GPU: Fix Edge Cases for TextureCopy](https://github.com/citra-emu/citra/pull/2809) by [wwylele](https://github.com/wwylele)
+
+The 3DS GPU has a data transfer mode called `TextureCopy`, which as the name says, is for copying textures but with a configurable gap, in case the texture is going to be copied into a smaller resolution area. This mode is enabled by setting the 3rd flag in the GPU and causes it to ignore every other flag except the 2nd, which is used to tell it whether or not to crop the texture. 
+
+What's nice about this is that it can be used as a quick and easy way for 3DS developers to duplicate textures and can be used in situations such as the one pictured running on hardware in Pokemon Super Mystery Dungeon below. This is really nice for those creating games, but when it came to running something that took advantage of this feature in Citra it didn't always work the same as it would on console. This provided us with an accuracy issue that needed to be solved.
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-before.png" />
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = a40e60c4bc7ab9c559ad6e3f1769b1128d68c8d5
+-->
+*How jagged*
+</p>
+
+Fortunately (and to much rejoice) wwylele stepped into the ring to wrestle with this issue. They prepared a [test program](https://github.com/wwylele/ctrhwtest/tree/master/texture-copy-test) to help gain an understanding of how hardware handles the TextureCopy operation in comparison to Citra.
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-after.png" />
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = 57438cf51f3bdae71a694d2ebe96ad563968781c
+-->
+Deoxys is having a bit of a hard time, no?
+</p>
+
+
+## [Citra-QT: UI Themes](https://github.com/citra-emu/citra/pull/2804) by [Kloen](https://github.com/kloen)
+
+[Kloen](https://github.com/kloen) has put the time and work into Citra's Qt frontend to make it themable! Now users can enjoy a dark mode and other custom coloring schemes!
```

Comment:
`themable` -> `themeable`

`other custom coloring schemes`: Users can't create their own custom color schemes through the config file, correct? Shouldn't we change it to "and `eventually` other custom coloring"?